firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -56,6 +59,13 @@
 /** A set of helper methods for tests */
 public class IntegrationTestUtil {
 
+  // Whether the integration tests should run against a local Firestore emulator instead of the
+  // Production environment. Note that the Android Emulator treats "10.0.2.2" as its host machine.
+  private static final boolean CONNECT_TO_EMULATOR = false;
+
+  private static final String EMULATOR_HOST = "10.0.2.2";
+  private static final int EMULATOR_PORT = 8081;
```

Comment:
FWIW, I think @allisonbm92 intended us to have multiple FirestoreProvider implementations representing different test environments.  That's how it was before the migration to github.  I don't have strong feelings about this, but we may want to clean this up at some point and either follow the FirestoreProvider pattern, or just ditch FirestoreProvider entirely.